src/Query/Builder.php

```diff
@@ -1158,11 +1158,6 @@ protected function compileWheres(): array
         return $compiled;
     }
 
-    /**
-     * @param  array $where
-     *
-     * @return array
-     */
```

Comment:
This change and `User.php` below seem unrelated to docs. How did these changes make it into the PR?

Reply:
I think this was rebased in by the phpcbf GH action, but not really sure as I've never explicitly changed or committed the file.

Same change from another PR I branched from 4.1:
https://github.com/mongodb/laravel-mongodb/pull/2764/files#diff-bdf045bbd1faa398417a788236b3b7da49c553bc4c6e7d6403568ecd775d2034L1161-L1165

Let me know how best to proceed here.

Reply:
Yes, this cs fixes are automatically committed by the GHA. The PR needs to be rebased as the fix has been applied to the [branch 4.1](https://github.com/mongodb/laravel-mongodb/commits/4.1/).

Reply:
Strange, when I used the GH button "Sync Fork" on this branch, it merged 4.2 into the branch. I'll revert that merge and perform the 4.1 merge manually.